### PR TITLE
[GLOSSARY] hash

### PR DESCRIPTION
**Pull request by MaxHillebrand**
adds cryptographic hash function to glossary

### CODE REVIEW

**Comment by RiccardoMasutti** [comment]
NACK.
I think Bitcoin basics should not be in Wasabi glossary.

File: docs/glossary/Glossary.md [hunk 1]
```diff
@@ -51,6 +51,15 @@ A hardware wallet is a special type of Bitcoin wallet which generates and stores
 Read more: [Hardware Wallet FAQs](/FAQ/FAQ-UseWasabi.md#hardware-wallet)
 :::
 
+:::details
+### Hash
+
+A cryptographic hash function takes any input of arbitrary size [the message] and computes a value of fixed size that is unique to the input.
```
Comment (yahiheb) [inline docs/glossary/Glossary.md]:
```suggestion
A cryptographic hash function takes any input of arbitrary size [the message] and computes a value of fixed size that is unique to the input, which is called a hash or a digest.
```

File: docs/glossary/Glossary.md [hunk 2]
```diff
@@ -51,6 +51,15 @@ A hardware wallet is a special type of Bitcoin wallet which generates and stores
...
+If one bit of the input changes, the output will be completely different, there are no two messages that result in the same hash [collision resistance].
```
[the remark this answers is not in the document]
Comment (MaxHillebrand) [reply docs/glossary/Glossary.md]:
```suggestion
If one bit of the input changes, the output will be completely different [avalanche effect].
And there are no two messages that result in the same hash [collision resistance].
```

**Comment by MaxHillebrand** [comment]
`Hash` is used at least 7 times in the docs, thus I think it should be included.

**Pull request merged**